rfcs/text/0000-extend-file-pe.md

```diff
@@ -0,0 +1,1076 @@
+# 0000: Extend the PE subfield of the File fieldset
```

Comment:
The "PE" field set actually appears under file, dll and process.

So we can simply call this:

```suggestion
# 0000: Extend the PE field set
```

or similar